### PR TITLE
Add math:log2/1

### PR DESCRIPTION
**Pull request by oliv3**

### CODE REVIEW

File: erts/emulator/beam/erl_math.c [hunk 1]
```diff
@@ -207,6 +207,46 @@ BIF_RETTYPE math_log_1(BIF_ALIST_1)
...
+}
+#endif
+
+BIF_RETTYPE math_log2_1(BIF_ALIST_1)
+{
+#ifdef HAVE_LOG2
+    return math_call_1(BIF_P, log2, BIF_ARG_1);
+#else
+    return math_log2(BIF_P, BIF_ARG_1);
+#endif
```
Comment (zuiderkwast) [inline erts/emulator/beam/erl_math.c]: The fallback function `math_log2()` looks much like copied code from `math_call_1()`. Why not define the C function `log2` if it does not exist? (Or if log2 can't be defined for some reason, use another name and `#define LOG2_FUNC log2_fallback`, etc.)

```C
#ifndef HAVE_LOG2
log2(double x)
{
    return log(x) / log(2.0);
}
#endif

BIF_RETTYPE math_log2_1(BIF_ALIST_1)
{
    return math_call_1(BIF_P, log2, BIF_ARG_1);
}
```

**Comment by oliv3** [comment]
Updated according to the last comments

**Comment by zuiderkwast** [comment]
It looks as this PR was closed 2 days ago. Was the branch renamed or something? Github doesn't seem to realize what happened. Maybe you should reopen it or create a new PR..... (I don't know which is best.)

**Comment by oliv3** [comment]
It is now at https://github.com/erlang/otp/pull/576 with the modifications you suggested

**Comment by zuiderkwast** [comment]
OK, nice. Just to be clear: I'm not responsible for anything OTP; just curious to see this merged.

**Comment by oliv3** [comment]
Your comment made sense

**Comment by zuiderkwast** [comment]
It looks good to me.

**Pull request closed without merging**